google-cloud-pubsub-v1/.owlbot-manifest.json

```diff
@@ -65,28 +56,37 @@
     "snippets/schema_service/validate_message.rb",
     "snippets/schema_service/validate_schema.rb",
     "snippets/snippet_metadata_google.pubsub.v1.json",
-    "snippets/subscriber/acknowledge.rb",
-    "snippets/subscriber/create_snapshot.rb",
-    "snippets/subscriber/create_subscription.rb",
-    "snippets/subscriber/delete_snapshot.rb",
-    "snippets/subscriber/delete_subscription.rb",
-    "snippets/subscriber/get_snapshot.rb",
-    "snippets/subscriber/get_subscription.rb",
-    "snippets/subscriber/list_snapshots.rb",
-    "snippets/subscriber/list_subscriptions.rb",
-    "snippets/subscriber/modify_ack_deadline.rb",
-    "snippets/subscriber/modify_push_config.rb",
-    "snippets/subscriber/pull.rb",
-    "snippets/subscriber/seek.rb",
-    "snippets/subscriber/streaming_pull.rb",
-    "snippets/subscriber/update_snapshot.rb",
-    "snippets/subscriber/update_subscription.rb",
-    "test/google/cloud/pubsub/v1/publisher_paths_test.rb",
-    "test/google/cloud/pubsub/v1/publisher_test.rb",
+    "snippets/subscription_admin/acknowledge.rb",
+    "snippets/subscription_admin/create_snapshot.rb",
+    "snippets/subscription_admin/create_subscription.rb",
+    "snippets/subscription_admin/delete_snapshot.rb",
+    "snippets/subscription_admin/delete_subscription.rb",
+    "snippets/subscription_admin/get_snapshot.rb",
+    "snippets/subscription_admin/get_subscription.rb",
+    "snippets/subscription_admin/list_snapshots.rb",
+    "snippets/subscription_admin/list_subscriptions.rb",
+    "snippets/subscription_admin/modify_ack_deadline.rb",
+    "snippets/subscription_admin/modify_push_config.rb",
+    "snippets/subscription_admin/pull.rb",
+    "snippets/subscription_admin/seek.rb",
+    "snippets/subscription_admin/streaming_pull.rb",
+    "snippets/subscription_admin/update_snapshot.rb",
+    "snippets/subscription_admin/update_subscription.rb",
+    "snippets/topic_admin/create_topic.rb",
+    "snippets/topic_admin/delete_topic.rb",
+    "snippets/topic_admin/detach_subscription.rb",
+    "snippets/topic_admin/get_topic.rb",
+    "snippets/topic_admin/list_topic_snapshots.rb",
+    "snippets/topic_admin/list_topic_subscriptions.rb",
+    "snippets/topic_admin/list_topics.rb",
+    "snippets/topic_admin/publish.rb",
+    "snippets/topic_admin/update_topic.rb",
     "test/google/cloud/pubsub/v1/schema_service_paths_test.rb",
     "test/google/cloud/pubsub/v1/schema_service_test.rb",
-    "test/google/cloud/pubsub/v1/subscriber_paths_test.rb",
-    "test/google/cloud/pubsub/v1/subscriber_test.rb",
+    "test/google/cloud/pubsub/v1/subscription_admin_paths_test.rb",
+    "test/google/cloud/pubsub/v1/subscription_admin_test.rb",
+    "test/google/cloud/pubsub/v1/topic_admin_paths_test.rb",
+    "test/google/cloud/pubsub/v1/topic_admin_test.rb",
     "test/helper.rb"
   ],
   "static": [
```

Comment:
Rename the publisher/helpers.rb and subscriber/helpers.rb in the static section because you've renamed those files.